packages/kbn-search-connectors/lib/update_connector_scheduling.ts

```diff
@@ -7,35 +7,40 @@
  */
 
 import { ElasticsearchClient } from '@kbn/core/server';
+import { errors } from '@elastic/elasticsearch';
 import { i18n } from '@kbn/i18n';
 
-import { CONNECTORS_INDEX } from '..';
-
-import { ConnectorDocument, SchedulingConfiguraton } from '../types/connectors';
+import { SchedulingConfiguraton } from '../types/connectors';
+import { ConnectorsAPIUpdateResponse } from '../types/connectors_api';
 
 export const updateConnectorScheduling = async (
   client: ElasticsearchClient,
   connectorId: string,
   scheduling: SchedulingConfiguraton
 ) => {
-  const connectorResult = await client.get<ConnectorDocument>({
-    id: connectorId,
-    index: CONNECTORS_INDEX,
-  });
-  const connector = connectorResult._source;
-  if (connector) {
-    const result = await client.index<ConnectorDocument>({
-      document: { ...connector, scheduling },
-      id: connectorId,
-      index: CONNECTORS_INDEX,
+  try {
+    const result = await client.transport.request<ConnectorsAPIUpdateResponse>({
```

Comment:
Has the result changed, or is it still the same? I'm not sure if the functions calling this function expect a specific result.

Reply:
It's safe to change the return type, the caller doesn't care about the result as long as the promise is successful https://github.com/elastic/kibana/blob/964a9804ffe17b412b5056b1645daa57c86d430f/x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts#L151